server/service/microsoft_mdm.go

```diff
@@ -1891,7 +1891,8 @@ func GetContextItem(secTokenMsg *fleet.RequestSecurityToken, contextItem string)
 // GetAuthorizedSoapFault authorize the request so SoapFault message can be returned
 func (svc *Service) GetAuthorizedSoapFault(ctx context.Context, eType string, origMsg int, errorMsg error) *fleet.SoapFault {
 	svc.authz.SkipAuthorization(ctx)
-	logging.WithErr(ctx, ctxerr.Wrap(ctx, errorMsg, "soap fault"))
+	logging.WithLevel(ctx, level.Info)
```

Comment:
I don't know enough about windows MDM but is it guaranteed that all errors coming through here is client errors?